static/src/inline-svgs/contributions/coin.svg

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 16.0.1, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->

```

Comment:
Will do - I'm a bit surprised though, https://github.com/guardian/frontend/blob/master/docs/inline-svgs.md#image-preparation says that inline SVGs are run through SVGO anyway, but they don't seem to be?